src/lib/logger-types.ts

```diff
@@ -0,0 +1,83 @@
+export enum LogEventMethod {
+  DO = "do",
+  UNDO = "undo",
+  REDO = "redo"
+}
+
+export enum LogEventName {
+  CREATE_TILE,
+  COPY_TILE,
+  MOVE_TILE,
+  DELETE_TILE,
+
+  VIEW_SHOW_DOCUMENT,
+  VIEW_SHOW_COMPARISON_DOCUMENT,
+  VIEW_SHOW_TEACHER_NETWORK_COMPARISON_DOCUMENT,
+  VIEW_ENTER_FOUR_UP,
+  VIEW_ENTER_ONE_UP,
+  VIEW_FOUR_UP_RESIZED,
+  VIEW_SHOW_COMPARISON_PANEL,
+  VIEW_HIDE_COMPARISON_PANEL,
+  VIEW_SHOW_SUPPORT,
+  VIEW_GROUP,
+
+  CREATE_PERSONAL_DOCUMENT,
+  CREATE_LEARNING_LOG,
+
+  SHOW_WORK,
+  SHOW_TAB,
+  SHOW_TAB_SECTION,
+  HIDE_SOLUTIONS,
+  SHOW_SOLUTIONS,
+
+  GRAPH_TOOL_CHANGE,
+  DRAWING_TOOL_CHANGE,
+  TABLE_TOOL_CHANGE,
+  TEXT_TOOL_CHANGE,
+  DATAFLOW_TOOL_CHANGE,
```

Comment:
These types are still here. I was hoping they would go away. This is the subject of the Drawing Tool TODO comment that you deleted.  No need to fix now though.

Reply:
I eliminated the types, which seemed like the more important violation and hence why I thought it made sense to remove the comment:
```
type LoggableTileChangeEvent =  Optional<JXGChange, "operation"> |
                                SimpleTileLogEvent |
                                Optional<ITableChange, "action"> |
                                DataflowProgramChange;
```
I agree that the constants should eventually be dealt with as well, but I had to stop somewhere. 🤷